themes/custom_tmpl/templates/search/home.phtml

```diff
@@ -0,0 +1,46 @@
+<?
```

Comment:
True. I think it's mainly the embedded AJAX call that gives me pause as something that could get messed up all too easily... but I don't really see a better way to handle it. Hmm.